src/components/svg-wrapper.tsx

```diff
@@ -169,10 +169,11 @@ const SvgWrapper = () => {
             const { x, y } = getMousePosition(e);
             const { x: svgX, y: svgY } = pointerPosToSVGCoord(x, y, svgViewBoxZoom, svgViewBoxMin);
             const nodesInRectangle = findNodesInRectangle(graph.current, selectStart.x, selectStart.y, svgX, svgY);
+            const edgesInRectangle = findEdgesConnectedByNodes(graph.current, new Set(nodesInRectangle));
             if (!e.shiftKey) {
-                dispatch(setSelected(nodesInRectangle));
+                dispatch(setSelected([...nodesInRectangle, ...edgesInRectangle]));
```

Comment:
Could be oneline with: `dispatch(setSelected([...new Set([...(!e.shiftKey ? selected : []), ...nodesInRectangle, ...edgesInRectangle])]));`